packages/ui/src/components/Visualization/Custom/ContextMenu/NodeContextMenu.test.tsx

```diff
@@ -137,6 +144,20 @@ describe('NodeContextMenu', () => {
     expect(item).toBeInTheDocument();
   });
 
+  it('should render a Paste as next step item if canHaveNextStep and isCompatible is true ', () => {
+    nodeInteractions.canHaveNextStep = true;
+    // Mock the `usePasteStep` hook to return compatible state
+    (usePasteStep as jest.Mock).mockReturnValue({
+      isCompatible: true,
+    });
+
+    const wrapper = render(<NodeContextMenu element={element} />, { wrapper: TestWrapper });
+
+    const item = wrapper.getByTestId('context-menu-item-paste-as-next-step');
+
+    expect(item).toBeInTheDocument();
+  });
+
```

Comment:
Can we add a test to check that clicking on the duplicate context item calls duplicate?